docs/user-guide/attributes-table.md

```diff
@@ -113,10 +113,53 @@ In *Edit mode*, you can delete existing features following these steps:
 
 <img src="../img/delete-line-shape.gif" />
 
+Quick Filtering
+-------------------
+
+[MapStore](https://mapstore2.geo-solutions.it/mapstore/#/) attibute table allow you to quickly filtering the table records(features) by using the 
+filter field located at the top of each table column. 
+
+<p><img src="../img/attribute-table-quick-filter.png" style="max-width:620px;"/></p>
+
+To use attribute table quick filters, you simply have to type in the filter field with the filter data relate to the respected column. 
```

Comment:
"To use **the** attribute table quick filters ..." 

please reformulate it better